app/models/cart.rb

```diff
@@ -1,15 +1,16 @@
 class Cart < ApplicationRecord
-    has_many :line_items, dependent: :destroy
-    def add_product(product)
-        current_item = line_items.find_by(product_id: product.id)
-        if current_item
-        current_item.quantity += 1
-        else
-        current_item = line_items.build(product_id: product.id)
-        end
-        current_item
-    end
-    def total_price
-        line_items.to_a.sum { |item| item.total_price }
+  has_many :line_items, dependent: :destroy
```

Comment:
Add blank line between association and methods

Reply:
done